src/LineChart.ts

```diff
@@ -34,6 +34,7 @@ module n3Charts {
         ['container', Factory.Container, element[0]],
         ['x-axis', Factory.Axis, 'x'],
         ['y-axis', Factory.Axis, 'y'],
+        ['lines-container', Factory.Series.LineSeriesContainer, 'y'],
```

Comment:
Yeah, I don't really know how to proceed here... I mean if we have the line series directly at this level, we'll end up adding/removing series regarding of the options at this level too... And I don't really want to have a variable number of things...

My idea was too have an entity create as many line series as need, and be just one container from the outside. We would then have the same for any other element type (dots, areas, and any other series type — even weird ones).